system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -759,6 +777,55 @@ pub mod bridging {
 			>;
 	}
 
+	pub mod to_ethereum {
+		use super::*;
+
+		parameter_types! {
+			/// User fee for ERC20 token transfer back to Ethereum.
+			/// Configure the fee to max Balance so that it is disabled.
+			/// Sensible value was 2_750_872_500_000.
+			pub const DefaultBridgeHubEthereumBaseFee: Balance = Balance::MAX;
+			pub storage BridgeHubEthereumBaseFee: Balance = DefaultBridgeHubEthereumBaseFee::get();
+			pub SiblingBridgeHubWithEthereumInboundQueueInstance: Location = Location::new(
+				1,
+				[
+					Parachain(SiblingBridgeHubParaId::get()),
+					PalletInstance(system_parachains_constants::kusama::snowbridge::INBOUND_QUEUE_PALLET_INDEX),
+				]
+			);
+
+			/// Set up exporters configuration.
+			/// `Option<MultiAsset>` represents static "base fee" which is used for total delivery fee calculation.
+			pub BridgeTable: sp_std::vec::Vec<NetworkExportTableItem> = sp_std::vec![
+				NetworkExportTableItem::new(
+					EthereumNetwork::get(),
+					Some(sp_std::vec![Junctions::Here]),
```

Comment:
@claravanstaden iirc, `Junctions::Here` is for `remote_location_filter`, so  e.g. for `pallet_xcm::reserve_transfer_assets` only destination `Location(2, [GlobalConsensus(Ethereum)]` will work. And if anything else is added after `GlobalConsensus` (e.g. `Location(2, [GlobalConsensus(Ethereum), AccountId(XYZ)]`) it won't be matched and allowed for routing.  
So do you expect here just ``Location(2, [GlobalConsensus(Ethereum)]`` or anything else?
If anything else, I would suggest to add here simple unit-tests (like this [one](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/xcm/xcm-builder/src/universal_exports.rs#L630)) to match several expected locations.

Reply:
> And if anything else is added after GlobalConsensus (e.g. Location(2, [GlobalConsensus(Ethereum), AccountId(XYZ)]) it won't be matched and allowed for routing.

Correct, this is the expected behaviour.

> So do you expect here just Location(2, [GlobalConsensus(Ethereum)] or anything else?

Just `Location(2, [GlobalConsensus(Ethereum)]` here. I added some tests anyway. 😄 

Tests added:
- [AssetHub Kusama](https://github.com/polkadot-fellows/runtimes/blob/c4c2995ba409ea739c6219994cb343be5bb5418d/system-parachains/asset-hubs/asset-hub-kusama/tests/snowbridge.rs)
- [AssetHub Polkadot ](https://github.com/polkadot-fellows/runtimes/blob/c4c2995ba409ea739c6219994cb343be5bb5418d/system-parachains/asset-hubs/asset-hub-polkadot/tests/snowbridge.rs)